### DISCUSSION TITLE
Parcel TS Config Path in Library

### DISCUSSION THREAD

**Discussion by rmcarias** [parcel-bundler/parcel#7712] [Q&A]
I am trying to configure parcel with TS using this example:
```
{
  "extends": "@parcel/config-default",
  "transformers": {
    "*.{ts,tsx}": ["@parcel/transformer-typescript-tsc"]
  }
}
```
Question is, I have a folder level tsconfig.json that inherits a tsconfig.base.json
How do I tell parcel where to find my tsconfig? In BabelRc you can do this:
```
["ts-plugin", { 
   ...allmyoptions
}] 
```
But I don't see documentation for doing the same thing in parcel. Very frustrating because it keeps wanting to load it's own TS ```compilerOptions``` which aren't enough for my library.

**Comment by mischnic** [comment]
This is the logic for loading tsconfig: https://github.com/parcel-bundler/parcel/blob/v2/packages/utils/ts-utils/src/loadTSConfig.js

It finds the nearest tsconfig from the entrypoint and passes the result to tsc. I imagine that shoudl then also take into account [extends](https://www.typescriptlang.org/tsconfig#extends)
